README.md

```diff
@@ -18,9 +18,25 @@ A difference to existing projects like OpenBCI is that it will not be necessary
 
 ## Screenshots
 <table>
-        <tr>
-<td><img src = "https://user-images.githubusercontent.com/20669217/55556050-432cb580-5704-11e9-85dd-d223c16d7a38.png" height = "500" width="250"></td>
+        
+  <tr>
+  <td><img src = "/docs/images/start_screen_1.png" height = "500" width="250"></td>
+
+  <td><img src = "/docs/images/start_screen_2.png" height = "500" width="250"></td>  
+
+  <td><img src = "/docs/images/start_screen_3.png" height = "500" width="250"></td>
+  </tr>
+
+  <tr>
+  <td><img src = "/docs/images/home_screen.png" height = "500" width="250"></td>
+  <td><img src = "/docs/images/nav_menu.png" height = "500" width="250"></td>
```

Comment:
Add one more image here. Otherwise it looks like an empty cell when viewed.

Reply:
Sure. Should I add the spectrum activity screenshot?

Reply:
![Demo](https://user-images.githubusercontent.com/22871961/60240854-a5deb900-98cf-11e9-802c-05459a430815.gif)
How about this as demo.gif? I got this from one of the PRs

Reply:
Yeah you can add it.